jam/umbrel-app.yml

```
@@ -2,7 +2,7 @@ manifestVersion: 1
 id: jam
 category: Finance
 name: Jam
-version: "0.0.10"
+version: "0.1.0"
 tagline: A user-friendly UI for JoinMarket
```

NIT: tagline could be adjusted to what's at [jamapp.org](https://jamapp.org/)

```suggestion
tagline: Your sats. Your privacy. Your profit.
```